tests/test_virtual_chassis.py

```diff
@@ -0,0 +1,71 @@
+import time
+import json
+import redis
+import pytest
+from swsscommon import swsscommon
+
+class TestVirtualChassis(object):
+    def check_chassis_sysport(self, dvs):
+        # check if instance is connected to a chassis db
+        cfgFile = "/usr/share/sonic/virtual_chassis/default_config.json"
+        (ret, out) = dvs.runcmd("sonic-cfggen -v  DEVICE_METADATA.localhost.chassis_db_address -j %s" % cfgFile)
+        verifyChassDb = True
```

Comment:
```suggestion
        verifyChassisDb = True
```

Would use the full name here. :)

Reply:
ack